test/modules/core/lib/layer.spec.js

```diff
@@ -211,27 +211,27 @@ test('Layer#diffProps', t => {
   let layer = new SubLayer(LAYER_PROPS);
   t.doesNotThrow(() => testInitializeLayer({layer, onError: t.notOk}), 'Layer initialized OK');
 
-  layer.diffProps(new SubLayer(Object.assign({}, LAYER_PROPS)).props, layer.props);
+  layer._diffProps(new SubLayer(Object.assign({}, LAYER_PROPS)).props, layer.props);
```

Comment:
Again the underscores change. which seems to be adding more than needed to the PR, but you know what you are doing!